src/EditorFeatures/Core.Wpf/EditorFeaturesWpfResources.resx

```diff
@@ -180,4 +180,19 @@
   <data name="Interactive_host_process_platform" xml:space="preserve">
     <value>Interactive host process platform</value>
   </data>
+  <data name="Multiple_members_are_inherited" xml:space="preserve">
+    <value>Multiple members are inherited</value>
+  </data>
+  <data name="is_inherited" xml:space="preserve">
+    <value>is inherited</value>
+  </data>
+  <data name="Navigate_to_0" xml:space="preserve">
+    <value>Navigate to '{0}'</value>
+  </data>
+  <data name="_0_in_1" xml:space="preserve">
+    <value>'{0}' in '{1}'</value>
+  </data>
+  <data name="Multiple_members_are_inherited_on_line_0" xml:space="preserve">
+    <value>Multiple members are inherited on line {0}</value>
+  </data>
```

Comment:
💡 Would be good to have a comment that this resource is only used for UI Automation / accessibility